README.md

```diff
@@ -45,6 +45,18 @@ All files with the `.css` extension will now be processed by PostCSS.
 
 `jekyll-postcss` will cache your styles to avoid rebuilding when nothing has changed.
 
+Also note that your `.css` files still need to have a frontmatter (top part separated with `---`) for them to be processed by jekyll.
```

Comment:
Can you link to YAML frontmatter instead of the parenthesized bit and capitalize Jekyll?